spharpy/spherical.py

```diff
@@ -301,23 +303,28 @@ def renormalize(data, channel_convention, current_norm, target_norm, axis):
     shape[axis] = -1
 
     data_renorm = data.copy()
-    if current_norm == 'n3d':
-        if target_norm == "sn3d":
-            data_renorm *= n3d_to_sn3d_norm(orders).reshape(shape)
-        elif target_norm == "maxN":
-            data_renorm *= n3d_to_maxn(acn).reshape(shape)
-
+    # normalize to 'n3d'
```

Comment:
maybe it is obious, but I don't get why we need to reshape the data. maybe not related to this pr.